### ISSUE TITLE
pickAFile doesn't seem to be returning anything

### ISSUE THREAD

**Issue by pacutter** [gatech-csl/jes#112]
I have a student running the latest version of JES and the pickAFile function doesn't seem to be returning anything.  Any ideas what we should look at/try?

**Comment by mjguzdial** [comment]
I've never seen that.  What version of which operating system?  Have you tried picking a file that is definitely in the student's writable space?  I'm thinking about the problems of directories that the student doesn't have read-access to, or directories that are mounted via the network.


- Mark


________________________________

**Comment by pacutter** [comment]
Hi Mark,
I’ve never seen it before either.  He’s running Windows 10 and downloaded jes-5.020-windows-java-included.zip.  We tried going back and downloading JES 4.3, but that would close as soon as he tried to open it. We tried hard coding a file path to see if that would work, but we ran out of class time and I don’t think he had the path right.  There were some Chinese characters in the username part of the path.  Maybe it does have something to do with permissions on his directories.  pickAFile would open the file picker window just fine, but then didn’t return anything.  I will look into that more the next time I see him.

Thanks,
Pam

**Comment by mjguzdial** [comment]
Jython can choke on unicode. I wonder if that's what's going on.

 - Mark


________________________________

**Comment by pacutter** [comment]
Yes, that was it. The student created a new user name (with no Japanese characters) on his laptop and then he was able to get the pickAFile function to work just fine.
Thanks for thinking of that!

Pam
